docs/developer-docs/latest/setup-deployment-guides/configurations/optional/cronjobs.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Cron jobs - Strapi Developer Documentation
+description: …
+---
+
+<!-- TODO: update SEO -->
+
+# Cron jobs
+
+:::prerequisites
+The `cron.enabled` configuration option should be set to `true` in [the `./config/server.js` file](/developer-docs/latest/setup-deployment-guides/configurations/required/server.md).
+:::
+
+`cron` allows scheduling arbitrary functions for execution at specific dates, with optional recurrence rules. These functions are named cron jobs. `cron` only uses a single timer at any given time, rather than reevaluating upcoming jobs every second/minute.
+
+This feature is powered by the [`node-schedule`](https://www.npmjs.com/package/node-schedule) package.
+
+The `cron` format consists of:
+
+```
+
+*    *    *    *    *    *
+┬    ┬    ┬    ┬    ┬    ┬
+│    │    │    │    │    |
+│    │    │    │    │    └ day of week (0 - 7) (0 or 7 is Sun)
+│    │    │    │    └───── month (1 - 12)
+│    │    │    └────────── day of month (1 - 31)
+│    │    └─────────────── hour (0 - 23)
+│    └──────────────────── minute (0 - 59)
+└───────────────────────── second (0 - 59, OPTIONAL)
+
+```
+
+To define cron jobs and have them run at the required times:
+
+1. [Create](#creating-a-cron-job) the appropriate file.
+2. [Enable](#enabling-cron-jobs) the cron jobs in the server configuration file.
+
+::: tip
+Optionally, cron jobs can be directly created in the `cron.tasks` key of the [server configuration file](/developer-docs/latest/setup-deployment-guides/configurations/required/server.md).
+:::
+
+## Creating a cron job
+
+To define a cron job, create a file with the following structure:
+
+```js
+// path: ./config/src/cron-tasks.js
+
+module.exports = {
+  /**
+   * Simple example.
+   * Every monday at 1am.
+   */
+
+  '0 0 1 * * 1': ({ strapi }) => {
+    // Add your own logic here (e.g. send a queue of email, create a database backup, etc.).
+  },
+};
+```
+
+If the cron job is required to run based on a specific timezone, configure it like the following:
+
+```js
+module.exports = {
+   /**
+   * Cron job with timezone example.
+   * Every Monday at 1am for Asia/Dhaka timezone.
+   * List of valid timezones: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List
+   */
+
+  '0 0 1 * * 1': {
+    task: ({ strapi }) => { /* Add your own logic here */ },
+    options: {
+      tz: 'Asia/Dhaka',
+    },
+  },
+};
+```
+
+## Enabling cron jobs
+
+To enable cron jobs, set `cron.enabled` to `true` in the [server configuration file](/developer-docs/latest/setup-deployment-guides/configurations/required/server.md) and declare the jobs:
+
+```js
+// path: ./config/server.js
+
+const cronTasks = require("./src/cron-tasks");
```

Comment:
```suggestion
const cronTasks = require("./cron-tasks");
```

Reply:
Thanks @derrickmehaffy  :-)

But judging [from what I saw](https://github.com/strapi/strapi/tree/releases/v4/examples/getstarted/config) I thought we would just keep the ./config/ "root" for required files (i.e. those that exist "by default" like server.js, admin.js, middlewares.js…) and add other files in ./config/src/ or ./config/whatever/

That's why it's presented like this in the project structure docs:
![Screenshot 2021-11-02 at 14 59 49](https://user-images.githubusercontent.com/4233866/139870001-71f935ef-2925-4a92-b99e-7402ded856d3.png)

tl;dr Not a requirement to have them in ./config/src/ but more a recommandation/best practice. WDYT?
cc @alexandrebodin

Reply:
Thanks Alex. I suggest we just move the cron-tasks file to ./config as Derrick suggests, then, and keep things simple (but I keep a mental note of your suggestion 🙂 )

Reply:
We can iterate later when we get to documenting best practices instead. I agree for now lets keep it simple (KISS principle :laughing: )